src/main/java/com/matt/forgehax/asm/utils/remapping/MCPMappingLoader.java

```diff
@@ -22,7 +23,7 @@
   private final String baseSrgDir = "{mc_ver}";
   private final String baseMappingDir = "{mc_ver}/{channel}_{map_ver}";
   private final String baseMappingUrl =
-    "http://export.mcpbot.bspk.rs/mcp_{channel}/{map_ver}-{mc_ver}/mcp_{channel}-{map_ver}-{mc_ver}.zip";
+    "https://github.com/ModCoderPack/MCPMappingsArchive/raw/master/mcp_{channel}/{map_ver}-{mc_ver}/mcp_{channel}-{map_ver}-{mc_ver}.zip";
   private final String baseSrgUrl =
     "http://export.mcpbot.bspk.rs/mcp/{mc_ver}/mcp-{mc_ver}-srg.zip";
```

Comment:
Need to change `"http://export.mcpbot.bspk.rs/mcp/{mc_ver}/mcp-{mc_ver}-srg.zip"` to `"https://github.com/ModCoderPack/MCPMappingsArchive/raw/master/mcp/{mc_ver}/mcp-{mc_ver}-srg.zip"`

Then this should work.

Reply:
it didn't appear to need it to launch, but it's nice to have everything actually fixed. (instead of relying on the art of the bodge.)

Reply:
fixed. (Thanks to https://github.com/mmvanheusden )